cmd/helm/upgrade.go

```diff
@@ -29,30 +29,57 @@ import (
 )
 
 const upgradeDesc = `
-This command upgrades a release to a new version of a chart.
+This command upgrades a release to a specified version of a chart and/or updates chart values.
 
-The upgrade arguments must be a release and chart. The chart
-argument can be either: a chart reference('stable/mariadb'), a path to a chart directory,
-a packaged chart, or a fully qualified URL. For chart references, the latest
-version will be specified unless the '--version' flag is set.
+Required arguments are release and chart. The chart argument can be one of:
+ - a chart reference('stable/mariadb'); use '--version' and '--devel' flags for versions other than latest,
+ - a path to a chart directory,
+ - a packaged chart,
+ - a fully qualified URL.
 
-To override values in a chart, use either the '--values' flag and pass in a file
-or use the '--set' flag and pass configuration from the command line. To force string
-values in '--set', use '--set-string' instead. In case a value is large and therefore
-you want not to use neither '--values' nor '--set', use '--set-file' to read the
-single large value from file.
+To customize the chart values use any of
+ - '--values'/'-f' to pass in a yaml file holding settings,
+ - '--set' to provide one or more key=val pairs directly,
+ - '--set-string' to provide key=val forcing val to be stored as a string,
+ - '--set-file' to provide key=path to read a single large value from a file at path.
 
-You can specify the '--values'/'-f' flag multiple times. The priority will be given to the
-last (right-most) file specified. For example, if both myvalues.yaml and override.yaml
-contained a key called 'Test', the value set in override.yaml would take precedence:
+To edit or append to the existing customized values add the 
+ '--reuse-values' flag, otherwise any existing customized values are ignored.
+
+If no chart value arguments are provided on the command line, any existing customized values are carried
+forward. To revert to use only values provided in the chart, use the '--reset-values' flag.
```

Comment:
This sentence reads a little awkward. Can we change this to

> If you want to revert back to just the values provided in the chart, use the '--reset-values' flag.

Reply:
done